src/work4.py

```diff
@@ -0,0 +1,15 @@
+from itertools import groupby
+lines = '''
```

Comment:
0 lines between import and code

Reply:
still 0 lines between import and code and 1 in other place. review not addressed